modin/experimental/core/execution/dask/implementations/pandas_on_dask/io/io.py

```diff
@@ -62,7 +62,7 @@ class ExperimentalPandasOnDaskIO(PandasOnDaskIO):
 
     def __make_read(*classes, build_args=build_args):  # noqa: GL08
         # used to reduce code duplication
-        return type("", (DaskWrapper, *classes), build_args)._read
+        return type("", (DaskWrapper, *classes), build_args).read
```

Comment:
Why is this changed?

Reply:
So that all calls go through `read` function in which, if necessary, we can wait for the end of calculations in remote functions.